turbo.json

```diff
@@ -1,6 +1,5 @@
 {
   "$schema": "https://turborepo.org/schema.json",
-  "baseBranch": "origin/main",
```

Comment:
The property does not exist anymore. Should not have any impact (https://github.com/vercel/turbo/pull/1681#issuecomment-1212645270)